--- node_modules/@storybook/core-server/dist/cjs/build-dev.js	2021-11-29 17:55:34.592907162 +0200
+++ patches/build-dev.js	2021-11-29 17:54:26.797250334 +0200
@@ -141,7 +141,7 @@
   } // Get package name and capitalize it e.g. @storybook/react -> React
 
 
-  var packageName = name.split('@storybook/').length > 0 ? name.split('@storybook/')[1] : name;
+  var packageName = name.split('@storybook/').length > 1 ? name.split('@storybook/')[1] : name;
   var frameworkName = packageName.charAt(0).toUpperCase() + packageName.slice(1);
   (0, _outputStartupInformation.outputStartupInformation)({
     updateInfo: versionCheck,
@@ -201,4 +201,4 @@
 
     process.exit(1);
   }
-}
\ No newline at end of file
+}
