--- node_modules/@muban/storybook/dist/client/preview/render.js	2021-11-29 20:25:30.707977906 +0200
+++ patches/render.js	2021-11-29 20:24:57.644210934 +0200
@@ -5,10 +5,10 @@
 var global_1 = require("global");
 var rootElement = global_1.document.querySelector('#root');
 function renderMain(_a) {
-    var storyFn = _a.storyFn, args = _a.args, showMain = _a.showMain;
+    var storyFn = _a.storyFn, args = _a.storyContext.args, showMain = _a.showMain;
+
     var componentStory = storyFn(args);
     showMain();
-    var data = (componentStory.data ? componentStory.data : args) || {};
     rootElement.innerText = '';
     var container = global_1.document.createElement('div');
     rootElement.appendChild(container);
@@ -16,11 +16,11 @@
     if (componentStory.component) {
         var app = muban_1.createApp(componentStory.component);
         app.component.apply(app, (componentStory.appComponents || []));
-        app.mount(container, componentStory.template, data);
+        app.mount(container, componentStory.template, args);
     }
     else {
         // only render the basic template
-        container.innerHTML = [].concat(componentStory.template(data)).join('');
+        container.innerHTML = [].concat(componentStory.template(args)).join('');
     }
 }
 exports.default = renderMain;
